src/components/SectionsList/data.json

```diff
@@ -5,28 +5,28 @@
       "type": "info",
       "text": "A quick reference sheet that will help you understand how to comply with the Regulation of Gatherings Act (RGA)",
       "short": "cheat",
-      "link": "../../../../documents/cheatsheet.pdf"
+      "link": "../../../../../../cheatsheet.pdf"
```

Comment:
Perhaps rather use a relative link starting from the page root? For example `/static/cheatsheet.pdf`. WHat do you think @AliciadW?

Reply:
In the Protest Guide PDF the links/URLs are given as www.protestguide.org.za/<docname>
This is why I added the PDFs in this way. I don't know if there is a way to add them in a folder (i.e static or documents), but keep the URL as per the document?

Reply:
This is much better. :-)

Reply:
Still a bit unsure why you are not using the relative root path, i.e. a path that starts with `/`, so in this case `/static/cheatsheet.pdf`.

I might be missing something? :)

Reply:
With the above the link is always in relation to the root of the domain, so it doesn't break if you move where it is called from around.